zzzz_modular_occulus/maps/submaps/deepmaint_rooms/core/empty_meadow.dmm

```diff
@@ -0,0 +1,187 @@
+"Sq" = (/obj/structure/table/glass,/obj/item/stack/medical/advanced/bruise_pack,/obj/spawner/surgery_tool/low_chance,/obj/spawner/surgery_tool/low_chance,/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/tiled/white/brown_perforated,/area/deepmaint)
+"Sy" = (/mob/living/carbon/superior_animal/roach/nanite,/turf/simulated/floor/tiled/steel/brown_perforated,/area/deepmaint)
+"SF" = (/obj/machinery/door/airlock/glass_security{id_tag = "BrigFoyer"; layer = 2.8; name = "Brig"; req_access = list(63)},/turf/simulated/floor/tiled/dark/cargo,/area/deepmaint)
+"TH" = (/obj/machinery/door/blast/regular{density = 0; dir = 4; icon_state = "pdoor0"; opacity = 0},/obj/machinery/door/window/eastleft,/obj/machinery/door/window/westleft,/turf/simulated/floor/tiled/steel,/area/deepmaint)
+"TQ" = (/obj/machinery/door/airlock/freezer{icon_state = "door_locked"; locked = 1; name = "sterile airlock"},/turf/simulated/floor/tiled/steel/bar_flat,/area/deepmaint)
+"Ue" = (/obj/spawner/pack/machine,/obj/effect/floor_decal/industrial/warning{dir = 5},/turf/simulated/floor/tiled/dark/gray_platform,/area/deepmaint)
+"Ui" = (/obj/machinery/door/airlock/freezer{icon_state = "door_locked"; locked = 1; name = "Data Storage"},/turf/simulated/floor/tiled/white/violetcorener,/area/deepmaint)
+"UB" = (/obj/structure/sign/department/rnd{pixel_y = -32},/turf/simulated/floor/tiled/steel,/area/deepmaint)
+"UK" = (/obj/effect/decal/cleanable/dirt,/obj/spawner/tank,/turf/simulated/floor/tiled/white/brown_perforated,/area/deepmaint)
+"Vr" = (/obj/effect/decal/cleanable/dirt,/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/tiled/white/bluecorner,/area/deepmaint)
+"VH" = (/obj/structure/flora/pottedplant/random,/turf/simulated/floor/tiled/white/brown_perforated,/area/deepmaint)
+"Wa" = (/obj/effect/decal/cleanable/cobweb,/turf/simulated/floor/tiled/steel/bar_light,/area/deepmaint)
+"Wk" = (/obj/effect/window_lwall_spawn/smartspawnphoron,/turf/simulated/floor/tiled/white/cargo,/area/deepmaint)
+"WG" = (/obj/spawner/pack/junk_machine,/obj/effect/floor_decal/industrial/warning{dir = 9},/obj/machinery/light,/turf/simulated/floor/tiled/dark/gray_platform,/area/deepmaint)
+"WT" = (/obj/structure/table/steel,/obj/item/weapon/storage/fancy/cigarettes/homeless,/turf/simulated/floor/tiled/white,/area/deepmaint)
+"Xd" = (/obj/machinery/button/remote/blast_door{id = "deepmaint_core_computers1"; pixel_x = -28},/obj/spawner/rig_module,/turf/simulated/floor/tiled/dark,/area/deepmaint)
+"Xs" = (/obj/effect/decal/cleanable/dirt,/turf/simulated/floor/tiled/white/violetcorener,/area/deepmaint)
+"Xy" = (/obj/structure/salvageable/os/data,/turf/simulated/floor/tiled/steel,/area/deepmaint)
+"XE" = (/obj/machinery/blackbox_recorder{desc = "Impossible to divine in purpose, impenetrably boring in shape."; icon_state = "comm_server_off"; name = "unknown machine"},/obj/effect/floor_decal/industrial/warning{dir = 9},/turf/simulated/floor/tiled/dark/gray_platform,/area/deepmaint)
```

Comment:
You cant really have more then one of these in the game world. If theirs one in telecoms then this will be deleted for that one do to how the generation works.

Could be wrong!

Reply:
That is not how that works